data.sql

```diff
@@ -12,17 +12,39 @@ INSERT INTO species (name) VALUES
     ('Pokemon'),
     ('Digimon');
 
--- Modify animals based on name and assign species_id
-UPDATE animals SET species_id = (CASE
-    WHEN name LIKE '%mon' THEN (SELECT id FROM species WHERE name = 'Digimon')
-    ELSE (SELECT id FROM species WHERE name = 'Pokemon')
-END);
+-- Insert data into the 'vets' table
+INSERT INTO vets (name, age, date_of_graduation) VALUES
+    ('Vet William Tatcher', 45, '2000-04-23'),
+    ('Vet Maisy Smith', 26, '2019-01-17'),
+    ('Vet Stephanie Mendez', 64, '1981-05-04'),
+    ('Vet Jack Harkness', 38, '2008-06-08');
 
--- Modify animals and assign owner_id
-UPDATE animals SET owner_id = (CASE
-    WHEN name IN ('Agumon') THEN (SELECT id FROM owners WHERE full_name = 'Sam Smith')
-    WHEN name IN ('Gabumon', 'Pikachu') THEN (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
-    WHEN name IN ('Devimon', 'Plantmon') THEN (SELECT id FROM owners WHERE full_name = 'Bob')
-    WHEN name IN ('Charmander', 'Squirtle', 'Blossom') THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond')
-    WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
-END);
+-- Insert data into the 'specializations' table
+INSERT INTO specializations (vet_id, species_name) VALUES
+    (1, 'Pokemon'),
+    (3, 'Digimon'),
+    (3, 'Pokemon'),
+    (4, 'Digimon');
```

Comment:
👍 Great progress on your project! To further enhance it, here are a couple of recommendations:

- [ ] Regarding the `specializations` table, please ensure that you've updated it in the `schema.sql` file to reflect the required changes. Make sure it establishes the correct many-to-many relationship between `vets` and `species`. This is a critical part of the project's functionality, so double-check that it aligns with the project requirements.

- [ ] Additionally, make sure that you've populated the `specializations` table as per the project requirements. Ensure that it contains the necessary data to establish the relationships between vets and the species they specialize in. This step is essential for the accurate functioning of your application.

By addressing these points, you'll ensure that your project aligns with the required schema and data population, setting a strong foundation for your application's functionality. Keep up the good work! 👍📊🏥